Dockerfile

```diff
@@ -5,10 +5,15 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 # Install base image and its dependencies
-FROM python:3.8-slim-buster
-# hadolint ignore=DL3008, DL3013, DL3015
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY requirements.txt /code/
```

Comment:
One disadvantage of moving `requirements.txt` file upfront is that if this file changes, the `autoconf` _etc_ will have to be re-installed anew during rebuilds, which will take a lot of time. (And could bring different versions of system packages along from remote repositories.) 

But I guess it is the price to pay if we want to make the images as slim as possible, so we should be OK...  Moreover we don't change it often.